lib/readSnapshots-request.js

```diff
@@ -0,0 +1,49 @@
+module.exports = ReadSnapshotsRequest;
```

Comment:
This filename should be "read-snapshots-request.js". The convention in sharedb is that all filenames are lowercase and all multi-word names are dash separated.